examples/hysteresis.ipynb

```diff
@@ -0,0 +1,408 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "id": "9296a9ce-b102-42e6-8499-5293e03ebc69",
+   "metadata": {},
+   "source": [
+    "# Hysteresis loop simulation"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "id": "d6761492-aa8d-44a7-bec8-fec0274ce023",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import mammos_entity as me\n",
+    "import mammos_units as u\n",
+    "import pandas as pd\n",
+    "from mammos_mumag.materials import Materials\n",
+    "from mammos_mumag.mesh import CUBE_20_nm\n",
+    "from mammos_mumag.parameters import Parameters\n",
+    "from mammos_mumag.simulation import Simulation"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "abd2375e-964e-4aa9-8e1e-ad4260354f71",
+   "metadata": {},
+   "source": [
+    "## Inputs"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "d1a49026-5a60-4448-92b0-4acf0cf10c4f",
+   "metadata": {},
+   "source": [
+    "### Geometry and Mesh\n",
+    "\n",
+    "We load one of the meshes coming with `mammos_mumag`, `CUBE_20_nm`, defining a 20 nanometer-side cube. The cube is surrounded by a shell and non-magnetic material in between the material cube and the shell. This is done is order to calculate demagnetisation field which decays to zero at infinity. For details: Imhoff, J. F., et al. \"An original solution for unbounded electromagnetic 2D-and 3D-problems throughout the finite element method.\" IEEE Transactions on Magnetics 26.5 (1990): 1659-1661. "
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "2ba5632b-36ac-4a67-999c-a1d8fa73dd7f",
+   "metadata": {},
+   "source": [
+    "### Material parameters\n",
+    "\n",
+    "We define the material magnetic parameters separately for the 3 domains: the magnetic material, the non-magnetic material and the shell. Therefore the parameters strictly related to the magnetic material will be only defined on the first domain. The other domains, will have magnetic properties equal to zero."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 3,
+   "id": "be3ed47b-ae03-4a68-aedb-08cfed197362",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "mat = Materials(\n",
+    "    domains=[\n",
+    "        {  # cube\n",
+    "            \"theta\": 0.0,\n",
+    "            \"phi\": 0.0,\n",
+    "            \"K1\": me.Ku(2805163.0590566983, unit=u.J / u.m**3),\n",
+    "            \"K2\": me.Ku(0.0, unit=u.J / u.m**3),\n",
+    "            \"Ms\": me.Ms(1189556.9164459957, unit=u.A / u.m),\n",
+    "            \"A\": me.A(6.015043651663635e-12, unit=u.J / u.m),\n",
+    "        },\n",
+    "        {  # sphere\n",
```

Comment:
- The word `sphere` is new, so far this was called `non-magnetic material`.
- Am I supposed to know that it is a sphere? I don't find that obvious from the text.

Reply:
Good point! Lets not call it shpere; we can call it non-magnetic material.